src/Struct/AgenticCommerce/V1/PayPalCart.php

```diff
@@ -0,0 +1,167 @@
+<?php declare(strict_types=1);
+/*
+ * (c) shopware AG <info@shopware.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Shopware\PayPalSDK\Struct\AgenticCommerce\V1;
+
+use OpenApi\Attributes as OA;
+use Shopware\PayPalSDK\Struct\Struct;
+
+#[OA\Schema(
+    schema: 'paypal_agentic_commerce_v1_pay_pal_cart',
+    required: ['items', 'paymentMethod']
+)]
+class PayPalCart extends Struct
```

Comment:
does this need to be called `PayPalCart` or can we just call it `Cart`? Seems like an unnecessary prefix.